alf/utils/value_ops.py

```diff
@@ -30,13 +30,21 @@ def action_importance_ratio(action_distribution, collect_action_distribution,
         Caller has to save tf.name_scope() and pass scope to this function.
 
         Args:
-            action_distribution: Probability of actions under target policy.
-            collect_action_distribution: Probability of action from behavior
-                policy, used to sample actions for the rollout.
-            action: action taken during rollout.
+            action_distribution (nested tf.distribution): Distribution over
+                actions under target policy.
+            collect_action_distribution (nested tf.distribution): distribution
+                over actions from behavior policy, used to sample actions for
+                the rollout.
+            action (nested tf.distribution): possibly batched action tuple
+                taken during rollout.
             action_spec (nested BoundedTensorSpec): representing the actions.
-            clipping_mode: 1: double_sided [-x, +x] clipping for PPOLoss.
-            scope: the scope object returned by tf.name_scope(), set outside.
+            clipping_mode (string): mode for clipping the importance ratio.
+                'double_sided': clips the range of importance ratio into
+                    [1-x, 1+x], which is used by PPOLoss.
+                'capping': clips the range of importance ratio into
```

Comment:
I'd suggest to make importance_ratio_clipping have similar meaning into both cases, so that we can switching between two modes without changing importance_ratio_clipping. This means that for 'capping', we use min(1+importance_ratio_clipping, importance_ratio). And change `x' to importance_ratio_clipping to be clearer.

Reply:
Good point, done.